### ISSUE TITLE
Do you have x86 version?

### ISSUE THREAD

**Issue by heiyanmaster** [ORCA666/EVA2#6]
Do you have x86 version?

**Comment by heiyanmaster** [comment]
X86 shows compilation failed

**Comment by ORCA666** [comment]
@heiyanmaster  no, as i said in the readme, x86 wont work, this whole project is a poc u can use ur own syscalls that work on x86, my syscalls were from syswhispers2 which only apply for x64 archs, so to make it work on x86 u need to change the apis i used:
NtAllocateVirtualMemory - NtWriteVirtualMemory - NtCreateThreadEx and there asm representation .
use this: https://github.com/mai1zhi2/SysWhispers2_x86  to do so
